src/bijectors/scale.jl

```diff
@@ -0,0 +1,80 @@
+struct Scale{T, N} <: Bijector{N}
+    a::T
+end
+
+Scale(a::T; dim::Val{D} = Val(0)) where {T<:Real, D} = Scale{T, D}(a)
+Scale(a::A; dim::Val{D} = Val(N)) where {T, D, N, A<:AbstractArray{T, N}} = Scale{A, D}(a)
+
+(b::Scale)(x) = b.a .* x
```

Comment:
Is there a particular reason we sometimes use `@.` and sometimes do the dot operations explictly.

Reply:
I don't have a specific style that I follow for this. It's more a matter of how many dots I need to write out:)
I usually go with `@.` if I have more than 2 operations I want to fuse